### PR TITLE
Added: Products page with navigation bar and sidebar

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
Added Product Page, Navigation, Sidebar

Live link --> https://deploy-preview-1--shop-up.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,261 @@
...
+                        <h3 class="my-4">Hot Deals</h3>
+                        <div class="categories-section flex-row my-4">
+                            <div class="product mr-5">
+                                <img class="img-responsive" src="https://www.oberlo.com/media/1603954764-image059.png?w=1824&fit=max" alt="product">
+                                <div class="my-3">
+                                    <h4>Product Name</h4>
+                                    <p class="brand-name">Brand Name</p>
+                                    <div class="flex-align-center">
+                                        <span class="mr-3">
+                                            <label>
```
Comment (vishalg8454) [inline index.html:149]: Can you explain the use of `label` tag here?

Comment (pratik-kumar-sahu) [reply index.html:149]: i am hiding the actual checkbox and showing star icon for rating. That's why used label to keep both of them inside it.

Comment (vishalg8454) [reply index.html:149]: I do not think that this is the correct use-case for `label` tag.

Comment (pratik-kumar-sahu) [reply index.html:149]: ok this is working for me. I'll look into this Vishal thanks for reviewing

**Review by aamrapaliwandhre — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1,12 +1,261 @@
...
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="https://fuzz-ui.netlify.app/styles/FuzzUI.css" />
+        <link rel="stylesheet" href="./styles/ShopUp.css" />
+        <title>ShopUp</title>
+    </head>
+    <body>
+        <main class="main-body flex-column p-3">
+            <div class="navigation flex-j-space-a-center p-5">
```
Comment (Shailseen) [inline index.html]: You use semantic tag here like nav/header instead of div.

File: index.html [hunk 3]
```diff
@@ -1,12 +1,261 @@
...
+                    <ion-icon class="icon mx-3" name="heart-outline"></ion-icon>
+                    <ion-icon class="icon mx-3" name="cart-outline"></ion-icon>
+                    <ion-icon class="icon mx-3" name="notifications-outline"></ion-icon>
+                    <img class="avatar img-rounded avatar-sm mx-3" src="https://i.pravatar.cc/150?img=3" alt="avatar" />
+                    <ion-icon class="icon mx-3" name="options-outline"></ion-icon>
+                </div>
+            </div>
+
+            <div class="flex-row containers">
+                <section class="sidebar p-4">
```
Comment (Shailseen) [inline index.html:33]: here also you can use aside tag.

**Review by Shailseen — APPROVED** [review]
Need some small changes and all  lgtm. you can proceed to merge.

File: index.html (hunk 2 above)
Comment (pratik-kumar-sahu) [reply index.html]: Sure thanks

**Pull request merged**